core/loadpoint.go

```diff
@@ -1217,7 +1210,7 @@ func (lp *Loadpoint) updateChargeCurrents() {
 	lp.log.DEBUG.Printf("charge currents: %.3gA", lp.chargeCurrents)
 	lp.publish("chargeCurrents", lp.chargeCurrents)
 
-	if lp.charging() {
+	if lp.charging() && lp.phaseSwitchTimeoutElapsed() {
```

Comment:
Is this still required? It was originally introduced, to prevent evcc to override any potential changes to the currents that were applied right before the phase switch. However, that entire logic (dividing by 3 or setting to maxA) were removed, altogether with the previous original setting of 0A to disable.
So given all that, I think this delay has become superflous now.

Reply:
I would think the delay is necessary, as measuredPhases is reset when number of phases is switched.
Without the delay evcc will measure used phases before the actual phase switch is performed.

With the current nightly when switching from 1 to 3 (car supports only 2 phases), evcc will report 1 - 2 - 1 - 2 active phases. With that delay it should report 1 - 2 as expected.

Reply:
Added it again, but hopefully properly working this time. Without such a timeout phase switches from one to three phases will always look like that 
![image](https://github.com/evcc-io/evcc/assets/34029447/2ab1990e-a23d-455f-a985-02d92b098e6f)
(Diagram for activePhases)